### PR TITLE
Adding title sending when allying

### PR DESCRIPTION
**Pull request by svaningelgem**
Allowing a 'title' to be sent when allying/de-allying. This is something I saw in another plugin and is a very nice to have feature.
If you don't like the text to pop up, you can always just leave the new `ally.add_title` and `ally.remove_title` messages empty. (which might be a requested change to keep backwards compatibility?)

When being working on this, I saw a bit of duplicated code between the `sendMessage`/`sendMessageF`/`getMessage`/`getMessageF` and the new title-methods. So I cleaned this up in a way that shouldn't impact the performance in the slightest bit, but improves the readability a lot.

Please do let me know if I need to change anything.

Screenshots:
__Ally:__
image
__Neutral:__
image

### CODE REVIEW

File: src/main/java/com/booksaw/betterTeams/commands/team/KickCommand.java [hunk 1]
```diff
@@ -47,7 +47,7 @@ public CommandResponse onCommand(TeamPlayer teamPlayer, String label, String[] a
 		team.removePlayer(player);
 
 		if (player.isOnline()) {
-			MessageManager.sendMessageF((CommandSender) player, "kick.notify", team.getName());
+			MessageManager.sendMessage((CommandSender) player, "kick.notify", team.getName());
 
 			if (Main.plugin.getConfig().getBoolean("titleRemoval")) {
 				player.getPlayer().sendTitle(" ", MessageManager.getMessage("kick.title"), 10, 100, 20);
```
Comment (booksaw) [inline src/main/java/com/booksaw/betterTeams/commands/team/KickCommand.java:53]: Could you switch existing places there is a title message to the new system

**Review by booksaw — CHANGES_REQUESTED** [review]
Thanks for making this, it may also be worth making the title message configurable unique to the chat message in case the user wants to only provide a short title message. But it may be a bit annoying to do, so don't worry if you don't want to

File: src/main/java/com/booksaw/betterTeams/commands/team/KickCommand.java (hunk 1 above)
Comment (svaningelgem) [reply src/main/java/com/booksaw/betterTeams/commands/team/KickCommand.java:53]: I saw this, but the current system is using the subtitle only, not the title. And I was wondering how I could integrate it without rehauling the messaging manager totally 🤣.

Do you have an idea on how to approach this?

**Comment by svaningelgem** [comment]
> Thanks for making this, it may also be worth making the title message configurable unique to the chat message in case the user wants to only provide a short title message.

Done. It wasn't much effort.

**Review by booksaw — APPROVED** [review]
Thanks for the contribution!

**Pull request merged**